example/requirements.txt

```diff
@@ -0,0 +1 @@
+kannon
```

Comment:
I think `fire` also should be install by user, but actually now it's done by kannon. I made #21